i18n/fi.po

```diff
@@ -0,0 +1,1222 @@
+"                    <a href=\"%(special)s\">special functions</a>, ..."
+msgstr ""
+"\n"
+"                    <a href=\"%(trig)s\">trigonometriset</a>,\n"
+"                    <a href=\"%(hyperbolic)s\">hyperboliset</a>,\n"
+"                    <a href=\"%(exponential)s\">eksponentit</a>,\n"
+"                    <a href=\"%(root)s\">juuret</a>,\n"
+"                    <a href=\"%(logarithm)s\">logaritmit</a>,\n"
+"                    <a href=\"%(abs)s\">itseisarvot</a>,\n"
+"                    <a href=\"%(spherical)s\">harmoniset pallofunktiot</a>,\n"
+"                    <a href=\"%(factorial)s\">kertomat ja gammafunktiot</"
+"a>,\n"
+"                    <a href=\"%(zeta)s\">zeta-funktiot</a>,\n"
+"                    <a href=\"%(polynomial)s\">polynomit</a>,\n"
+"                    <a href=\"%(special)s\">erikoisfunktiot</a>, ..."
+
+#: templates/features.html:46
+msgid "Substitution:"
+msgstr "Sijoitus:"
+
+#: templates/features.html:46
+msgid "example"
+msgstr "esimerkki"
+
+#: templates/features.html:47
+msgid "Numbers:"
+msgstr "Numerot:"
+
+#: templates/features.html:48
+#, python-format
+msgid ""
+"\n"
+"                    <a href=\"%(integer)s\">arbitrary precision integers</"
+"a>,\n"
+"                    <a href=\"%(rational)s\">rationals</a>,\n"
+"                    <a href=\"%(float)s\">and floats</a>\n"
+"                    "
+msgstr ""
+"\n"
+"                    <a href=\"%(integer)s\">mielivaltaisen tarkat "
+"kokonaisluvut</a>,\n"
+"                    <a href=\"%(rational)s\">rationaaliluvut</a>,\n"
+"                    <a href=\"%(float)s\">ja liukuluvut</a>\n"
+"                    "
+
+#: templates/features.html:58
+msgid "Noncommutative symbols"
+msgstr "Epävaihdannaiset"
+
+#: templates/features.html:59
+msgid "Pattern matching"
+msgstr "Kuvion täsmäys"
+
+#: templates/features.html:65
+msgid "Polynomials"
+msgstr "Polynomit"
+
+#: templates/features.html:69
+msgid " division, gcd, ... "
+msgstr "jaollisuus, syt, ..."
+
+#: templates/features.html:70
+msgid "Factorization"
+msgstr "Tekijöihin jakaminen"
+
+#: templates/features.html:71
+msgid "Square-free decomposition"
+msgstr "Neliötön hajotelma"
+
+#: templates/features.html:72
+msgid "Gröbner bases"
+msgstr "Gröbnerin kannat"
+
+#: templates/features.html:73
+msgid "Partial fraction decomposition"
+msgstr "Osamurtohajotelma"
+
+#: templates/features.html:74
+msgid "Resultants"
+msgstr "Resultantit"
+
+#: templates/features.html:80
+msgid "Calculus"
+msgstr "Differentiaalilaskenta"
+
+#: templates/features.html:82
+msgid "Integration Documentation"
+msgstr "Integrointidokumentit"
+
+#: templates/features.html:85
+msgid "Limits:"
+msgstr "Raja-arvot"
+
+#: templates/features.html:89
+msgid "Differentiation"
+msgstr "Derivointi:"
+
+#: templates/features.html:92
+msgid "Integration:"
+msgstr "Integrointi:"
+
+#: templates/features.html:93
+msgid "It uses extended Risch-Norman heuristic"
+msgstr "Se käyttää laajennettua Risch-Norman - heuristiikkaa"
```

Comment:
Here and elsewhere, [note grammar for space before/after dash](http://www.kielikello.fi/index.php?mid=2&pid=11&aid=450): `Risch-Norman-heuristiikka`, `Planet Sympy -blogi`